doc/source/timeseries.rst

```diff
@@ -703,6 +703,23 @@ regularity will result in a ``DatetimeIndex``, although frequency is lost:
 
    ts2[[0, 2, 6]].index
 
+.. _timeseries.iterating:
```

Comment:
can you add a reference to the groupby.iterating section (and a reverse ref from there to here)

Reply:
@jreback 

I didn't find how to add a reference in the [docs](https://pandas.pydata.org/pandas-docs/stable/contributing.html). Could you share how to do that?